openedx/core/djangoapps/content/learning_sequences/api/outlines.py

```diff
@@ -165,6 +166,7 @@ def get_course_outline(course_key: CourseKey) -> CourseOutlineData:
 
         sequence_data = CourseLearningSequenceData(
             usage_key=sequence_model.usage_key,
+            usage_key_hash=hash_usage_key(sequence_model.usage_key),
```

Comment:
Shouldn't this come straight from the model?

Reply:
Doi, yes. Thanks.